app/code/core/Mage/Directory/Model/Currency/Import/Fixerio.php

```diff
@@ -36,7 +36,7 @@ class Mage_Directory_Model_Currency_Import_Fixerio extends Mage_Directory_Model_
      *
      * @var string
      */
-    protected $_url = 'https://api.apilayer.com/fixer/latest?apikey={{ACCESS_KEY}}&base={{CURRENCY_FROM}}&symbols={{CURRENCY_TO}}';
+    protected $_url = 'http://data.fixer.io/api/latest?access_key={{ACCESS_KEY}}&base={{CURRENCY_FROM}}&symbols={{CURRENCY_TO}}&format=1';
```

Comment:
Why only http?

Reply:
https seems to work so it should be https

Reply:
HTTPS requires a paid account as far as I know, mostly legacy implementations just use 30 requests a month (one a day) and do not require a paid account, but we could make this configurable.